src/main/java/org/jboss/pnc/bifrost/endpoint/LogUpload.java

```diff
@@ -93,6 +106,34 @@ public String uploadFinalLog(@Valid FinalLogUpload logUpload, @Context HttpHeade
         return "ok";
     }
 
+    @Path("/{processContext}/delete")
+    @DELETE
+    @RolesAllowed("**") // FIXME change to specific allowed roles
+    public Response deleteFinalLog(@PathParam("processContext") String processContext) {
+        // parse process context
+        long processContextLong;
+        if (processContext.startsWith("build-")) {
+            processContextLong = idConverter.convert(processContext);
+        } else {
+            try {
+                processContextLong = Long.parseLong(processContext);
```

Comment:
The process context is basically the buildId, string encoded. So I think this will break.

We'll have to figure out exactly what is processContext since in my bifrost tests, process context = build id.

Reply:
you're correct, I've found out that the IdConverted nicely handles all cases `Long, Base, Base with build- prefix`!